nicenumber/nicenumber.py

```diff
@@ -166,6 +166,36 @@ def to_color(number:int, color:list = None):
     -------
     str
         String number with different colors
+
+    Examples
+    ----------
+    >>> print(to_color(13637373737348738787, ['Yellow', 'Red']))
     """
+    
```

Comment:
Could add in a check to give user a better error message if they pass in a list of colors which aren't in the palette eg: 
```
to_colour(123456, ['limegreen', 'pink'])

>>> ValueError: "Colors not available in palette. Available options are: ... <print list of palette colors> ... "
```

```suggestion
if not all(item in palette for item in color):
    raise ValueError(f'Colors not available in palette. Available options: {list(palette.keys())}')
```